lightbluetent/api.py

```diff
@@ -206,10 +243,21 @@ def is_meeting_running(self, params):
 
     def request(self, call, params):
         url = self.build_url(call, params)
-        res = requests.get(url, timeout=(0.5, 10))
+
+        try:
+            res = requests.get(url, timeout=(0.5, 10))
+
+        except requests.exceptions.ReadTimeout:
+            current_app.logger.info(f"Timeout timed out! Requests.exceptions.ReadTimeout when making API call { call }")
+            return None
+
```

Comment:
Before converting to XML here, we should probably also verify the actual HTTP code of the response object, like the status code: https://requests.readthedocs.io/en/master/. Though xmltodict might already do some verification here.

Reply:
Actually I don't believe it does, as xmltodict just converts the xml and has nothing to do with requests.